### PR TITLE
Milestone 8: Details modal

### PR DESCRIPTION
**Pull request by BenMKT**
For this milestone: 
- Implemented a popup window that includes the project details on both desktop and mobile.
- When the user clicks the button to check project details, the popup with details about the project appears.
- When the user clicks the close button, the popup disappears.
- In order to associate each project with the popup window details, refactored the project section.
- Updated the main section so that the projects are created dynamically using the information stored in the JavaScript object.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -19,3 +19,241 @@ navLink.forEach((navLink) => {
...
+            <li>${project.skills[0]}</li>
+            <li>${project.skills[1]}</li>
+            <li></>${project.skills[2]}</li>
+          </ul>
+          <button id='prjbtn' name='prjbtn' data-modal-target='#prjbtn'>See Project</button>
+        </div>
+      </div>`;
+});
+
+const popupContainer = document.querySelector('.popupContainer');
```
Comment (victorokoroji) [inline script.js:134]: - [x] Kindly ensure that the mobile view of your design matches the figma design. 

figma design 👇🏽 
image

your design 👇🏽 
image

Comment (victorokoroji) [inline script.js:134]: - [x] I also noticed that you have a horizontal scroll in your popup menu which is not a good user experience. kindly fix it 👍🏽 
image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @BenMKT, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- Good popup function ✅ 
- The PR title and description are professional and descriptive ✅ 
- Good UI design ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (BenMKT) [reply script.js:134]: - Added an `overflow-x: hidden` property to resolve the horizontal scroll bug.
- Increased `left % margin` to `.poplistbtn` to resolve the spacing bug.
- Removed the `height` property from popup `.body1` to reduce huge padding-bottom spacing.

Comment (BenMKT) [reply script.js:134]: - Added an overflow-x: hidden property to resolve the horizontal scroll bug.
- Increased left % margin to .poplistbtn to resolve the spacing bug.
- Removed the height property from popup .body1 to reduce huge padding-bottom spacing.

**Review by usorfaitheloho — APPROVED** [review]
Hi @BenMKT  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**